File: Pemrograman 1/MODUL 1/Bahasa Python/PRAK110-2410817110002-Akhmad_Daffa_Azzikri.py
import math

Alas = 5
Tinggi = 12
Sisi_A = 12
Sisi_C = 5
B =(Sisi_A*Sisi_A) + (Sisi_C*Sisi_C)
Sisi_B = math.sqrt(B)
hasil_keliling = Sisi_A + Sisi_B + Sisi_C
hasil_luas = 1/2 * Alas * Tinggi


print("Diketahui :\nAlas = %d cm \nTinggi = %d cm\n"%(Alas, Tinggi))
print("Jawab :\nSisi A = %d cm\nSisi B = %.0f cm\nSisi C = %d cm"%(Sisi_A,Sisi_B,Sisi_C))
print("Keliling = %.0f cm"%(hasil_keliling))
print("Luas = %.0f cm"%(hasil_luas))

File: Pemrograman 1/MODUL 1/Bahasa C/PRAK110-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>
#include <math.h>

int main(){
    int alas = 5;
    int tinggi = 12;
    int sisi_A = 12, sisi_C = 5;
    double B = (sisi_A*sisi_A) + (sisi_C*sisi_C);
    double sisi_B = sqrt(B);


    printf("Diketahui :\nAlas = %d cm \nTinggi = %d cm\n\n", alas, tinggi);
    printf("Jawab :\nSisi A = %d cm\nSisi B = %.0f cm\nSisi C = %d cm\n", sisi_A, sisi_B, sisi_C);
    printf("Keliling = %.0f cm\n", sisi_A + sisi_B + sisi_C);
    printf("Luas = %.0f cm", 1/2 * alas * tinggi);
    return 0;
}